examples/run_FS.py

```diff
@@ -32,148 +32,163 @@
 case and 4 times in the second case.  for nScans = 2, the
 SpinCore will trigger 4 times in the first case and 8 times
 in the second case.
-'''
-#}}}
+"""
+# }}}
 from pyspecdata import *
-from numpy import *
-from . import SpinCore_pp 
-import socket
-import sys
-import time
-
-#{{{ for setting EPR magnet
-def API_sender(value):
-    IP = "jmfrancklab-bruker.syr.edu"
-    if len(sys.argv) > 1:
-        IP = sys.argv[1]
-    PORT = 6001
-    print("target IP:", IP)
-    print("target port:", PORT)
-    MESSAGE = str(value)
-    print("SETTING FIELD TO...", MESSAGE)
-    sock = socket.socket(socket.AF_INET, # Internet
-            socket.SOCK_STREAM) # TCP
-    sock.connect((IP, PORT))
-    sock.send(MESSAGE)
-    sock.close()
-    print("FIELD SET TO...", MESSAGE)
-    time.sleep(3)
-    return
-#}}}
+from pylab import *
+import os
+import SpinCore_pp
+from SpinCore_pp.ppg import run_spin_echo
+from datetime import datetime
+from Instruments.XEPR_eth import xepr
+import numpy as np
+import h5py
 
-field_axis = linspace(3490.0,3510.0,5,endpoint=False)
+field_axis = r_[3490.0:3510.0:1.0]
 fl = figlist_var()
-date = '190614'
-output_name = 'FS_1'
-adcOffset = 34 
-carrierFreq_MHz = 14.894439
-tx_phases = r_[0.0,90.0,180.0,270.0]
-amplitude = 1.0
-nScans = 1
-nEchoes = 1
-phase_cycling = True 
+# {{{importing acquisition parameters
+config_dict = SpinCore_pp.configuration("active.ini")
+# }}}
+# {{{create filename and save to config file
+date = datetime.now().strftime("%y%m%d")
+config_dict["type"] = "field"
+config_dict["date"] = date
+config_dict["field_counter"] += 1
+filename = f"{config_dict['date']}_{config_dict['chemical']}_{config_dict['type']}"
+# }}}
+print("Here is my field axis:",field_axis)
+#{{{ phase cycling
+phase_cycling = True
 if phase_cycling:
-    nPhaseSteps = 8
+    nPhaseSteps = 4
```

Comment:
we are using the default ph1 with 4 steps in the spin_echo ppg so we don't need to specify "ph1 = r_[0,1,2,3]"